static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js

```diff
@@ -0,0 +1,157 @@
+define([
+    'bean',
+    'fastdom',
+    'qwery',
+    'common/utils/$',
+    'common/utils/storage',
+    'common/utils/config',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/mediator',
+    'common/modules/onward/history',
+    'common/utils/ajax',
+    'text!common/views/experiments/recommended-for-you.html',
+    'inlineSvg!svgs/icon/profile-36',
+    'inlineSvg!svgs/icon/arrow-right',
+    'inlineSvg!svgs/icon/marque-36',
+    'common/utils/fetch'
+], function (
+    bean,
+    fastdom,
+    qwery,
+    $,
+    storage,
+    config,
+    template,
+    svg,
+    mediator,
+    history,
+    ajax,
+    recommendedForYouTemplate,
+    profileIcon,
+    rightArrowIcon,
+    guardianLogo,
+    fetch
+) {
+    return function () {
+        this.id = 'RecommendedForYouRecommendations';
+        this.start = '2016-08-02';
+        this.expiry = '2016-12-23';
+        this.author = 'Joseph Smith';
+        this.description = 'Add a personalised container to fronts';
+        this.audience = 1;
```

Comment:
I would set this to 0 initially so that we can deploy it live and test it by force-opting into the test without opting in any real users